README.md

```diff
@@ -55,7 +55,13 @@ Check out how the [Release Process](https://github.com/immutable/imx-core-sdk/#r
 
 ### Configuration
 
-A configuration object is required to be passed into Core SDK requests. This can be obtained by using the `getConfig` function available within the Core SDK. You are required to select the Ethereum network. The Immutable X platform currently supports `ropsten` for testing and `mainnet` for production.
+A configuration object is required to be passed into Core SDK requests. This can be obtained by using the `getConfig` function available within the Core SDK. You are required to provide the correct contract addresses and api base path of the network you wish to use. The Immutable X platform currently supports `ropsten` for testing and `mainnet` for production.
```

Comment:
Would it be needed to specify in the text the chain ID as well as the contract addresses and API path?

Reply:
Good point. Will update. Hoping we get that new api/contractaddresses endpoint so all this can be deleted.

Reply:
[9924a12](https://github.com/immutable/imx-core-sdk/pull/121/commits/9924a12ec7f6f5b3bfb05a61ca7dbac572c43fbb)